cp ...System.XML.dll fails on mac

### Environment

-   Pythonnet version: master https://github.com/pythonnet/pythonnet/commit/f8e147b97112c526c6548e900dee0fc41cdb39b1
-   Python version: 3.7.0
-   Operating System: macOS 10.14.1

### Details

When compiling pythonnet from source on macOS, I noticed that this line:
https://github.com/pythonnet/pythonnet/blob/f8e147b97112c526c6548e900dee0fc41cdb39b1/src/runtime/Python.Runtime.15.csproj#L134

causes the build to fail:
```
       "/.../pythonnet/pythonnet.15.sln" (Build target) (1) ->
       "/.../pythonnet/src/runtime/Python.Runtime.15.csproj" (default target) (2) ->
       "/.../pythonnet/src/runtime/Python.Runtime.15.csproj" (Build target) (2:3) ->
       (BeforeBuild target) ->
         /.../pythonnet/src/runtime/Python.Runtime.15.csproj(134,5): error MSB3073: The command "cp /Users/ztl8702/.nuget/packages//microsoft.targetingpack.netframework.v4.5/1.0.1/lib/net45/System.XML.dll /Users/ztl8702/.nuget/packages//microsoft.targetingpack.netframework.v4.5/1.0.1/lib/net45/System.Xml.dll" exited with code 1.

    5 Warning(s)
    1 Error(s)
```

And when I ran:
```bash
cp /Users/ztl8702/.nuget/packages//microsoft.targetingpack.netframework.v4.5/1.0.1/lib/net45/System.XML.dll /Users/ztl8702/.nuget/packages//microsoft.targetingpack.netframework.v4.5/1.0.1/lib/net45/System.Xml.dll
```

in the Terminal, I got:
```
cp: .../net45/System.Xml.dll and .../net45/System.XML.dll are identical (not copied).
```

So it appears to me that macOS by default has [a case-insensitive filesystem](https://developer.apple.com/library/archive/documentation/FileManagement/Conceptual/APFS_Guide/FAQ/FAQ.html)

> APFS, like HFS+, is case-sensitive on iOS and is available in case-sensitive and case-insensitive variants on macOS, with **case-insensitive being the default**.
 
My hack was to check for existence of `System.Xml.dll` before `cp`. Maybe there is a more elegant way. (I actually wonder why the `cp` command is there in the first place)

--- a/src/runtime/Python.Runtime.15.csproj
+++ b/src/runtime/Python.Runtime.15.csproj
@@ -131,7 +131,7 @@
 
   <Target Name="BeforeBuild" Condition="'$(TargetFramework)'=='net40' AND $(Configuration.Contains('Mono')) AND '$(OS)' != 'Windows_NT'">
     <!--Endless war!-->
-    <Exec Command="cp $(NuGetPackageRoot)/microsoft.targetingpack.netframework.v4.5/1.0.1/lib/net45/System.XML.dll $(NuGetPackageRoot)/microsoft.targetingpack.netframework.v4.5/1.0.1/lib/net45/System.Xml.dll" />
+    <Exec Command="[[ -e $(NuGetPackageRoot)/microsoft.targetingpack.netframework.v4.5/1.0.1/lib/net45/System.Xml.dll ]] || cp $(NuGetPackageRoot)/microsoft.targetingpack.netframework.v4.5/1.0.1/lib/net45/System.XML.dll $(NuGetPackageRoot)/microsoft.targetingpack.netframework.v4.5/1.0.1/lib/net45/System.Xml.dll" />
   </Target>
   <Target Name="AfterBuild">
     <Copy Condition="'$(TargetFramework)'=='net40'" SourceFiles="$(TargetAssembly)" DestinationFolder="$(PythonBuildDir)" />

